_opensearch/search/sort.md

```diff
@@ -0,0 +1,880 @@
+GET students/_search
+{
+  "query": {
+    "match_all": {}
+  },
+  "sort": [
+    {
+      "average": {
+        "order": "desc"
+      }
+    }
+  ],
+  "track_scores": true
+}
+```
+
+## Sorting by geo distance
+
+You can sort documents by `_geo_distance`. The following parameters are supported:
+
+Parameter | Description
+:--- | :---
+distance_type | Specifies the method of computing the distance. Valid values are `arc` and `plane`. The `plane` method is faster but less accurate for long distances or close to the poles. Default is `arc`.
+mode | Specifies how to handle a field with several geopoints. By default, documents are sorted by the shortest distance when the sort order is ascending, and by the longest distance when the sort order is descending. Valid values are `min`, `max`, `median`, and `avg`.
+unit | Specifies the units to compute sort values. Default is meters (`m`).
+ignore_unmapped | Specifies how to treat an unmapped field. Set `ignore_unmapped` to `true` to ignore unmapped fields. Default is `false` (produce an error when encountering an unmapped field).
+
+The `_geo_distance` parameter does not support `missing_values`. The distance is always considered to be `infinity` when a document does not contain the field used for computing distance.
+{: .note}
+
+For example, index two documents with geopoints:
+
+```json
+PUT testindex1/_doc/1
+{
+  "point": [74.00, 40.71] 
+}
+
+PUT testindex1/_doc/2
+{
+  "point": [73.77, -69.63] 
+}
+```
+
+Search for all documents and sort them by the distance from the provided point:
+
+```json
+GET testindex1/_search
+{
+  "sort": [
+    {
+      "_geo_distance": {
+        "point": [59, -54],
+        "order": "asc",
+        "unit": "km",
+        "distance_type": "arc",
+        "mode": "min",
+        "ignore_unmapped": true
+      }
+    }
+  ],
+  "query": {
+    "match_all": {}
+  }
+}
+```
+
+The response contains the sorted documents:
+
+```json
+{
+  "took" : 864,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 2,
+      "relation" : "eq"
+    },
+    "max_score" : null,
+    "hits" : [
+      {
+        "_index" : "testindex1",
+        "_id" : "2",
+        "_score" : null,
+        "_source" : {
+          "point" : [
+            73.77,
+            -69.63
+          ]
+        },
+        "sort" : [
+          1891.2667493895767
+        ]
+      },
+      {
+        "_index" : "testindex1",
+        "_id" : "1",
+        "_score" : null,
+        "_source" : {
+          "point" : [
+            74.0,
+            40.71
+          ]
+        },
+        "sort" : [
+          10628.402240213345
+        ]
+      }
+    ]
+  }
+}
+```
+
+You can provide coordinates in any format supported by geopoint. For a description of all formats, see the [geopoint field type documentation]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point).
```

Comment:
Should "geopoint" be plural? Or should it be "supported by the geopoint field type"?